README.rst

```diff
@@ -800,32 +808,32 @@ For convenience, here is a simple example:
 
         )
 
-        ARGS = FUN.mapfiles(VAR.bindir, '/usr/sbin', bins) + [
+        ARGS.extend(FUN.mapfiles(VAR.bindir, '/usr/sbin', bins) + [
           'README.rst=/usr/share/doc/' + VAR.fullname '/',
-        ]
+        ])
 
 ``$P/client.pkg``:
 
 .. code:: py
 
-        from defaults import OPTS
+        import defaults
 
         bins = 'client clitool'
 
         OPTS.update(
 
           name = VAR.fullname,
 
-          description = FUN.desc(OPTS, 'tools', epilog='These are just ' +
+          description = FUN.desc('tools', epilog='These are just ' +
```

Comment:
Ditto comment about moving this change to the commit titled `Remove OPTS argument from FUN.desc()`.